### PR TITLE
Initial Upload

### PR DESCRIPTION
**Pull request by skiselev1**
Initial upload is done. Could you please review it?

Thanks,
Sergey

### CODE REVIEW

**Review by NovaVic — CHANGES_REQUESTED** [review]
Lots of cleanup to do.
Also source and destination of merge are not the right ones.

File: src/frontend/csrs-portal/package.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "@angular/cdk": "^12.2.7",
+    "@angular/common": "~12.2.7",
+    "@angular/compiler": "~12.2.7",
+    "@angular/core": "~12.2.7",
+    "@angular/forms": "~12.2.7",
+    "@angular/material": "^12.2.7",
+    "@angular/platform-browser": "~12.2.7",
+    "@angular/platform-browser-dynamic": "~12.2.7",
+    "@angular/router": "~12.2.7",
+    "@azure/ai-form-recognizer": "^3.1.0",
```
[the remark this answers is not in the document]
Comment (skiselev1) [reply src/frontend/csrs-portal/package.json]: Thank you! it was cleaned.

File: src/frontend/csrs-portal/package.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~12.2.7",
+    "@angular/cli": "~12.2.7",
+    "@angular/compiler-cli": "~12.2.7",
+    "@types/faker": "^5.5.7",
+    "@types/jasmine": "~3.8.2",
+    "@types/jasminewd2": "~2.0.10",
+    "@types/node": "^16.4.13",
+    "codelyzer": "^6.0.2",
+    "faker": "^5.5.3",
```
[the remark this answers is not in the document]
Comment (skiselev1) [reply src/frontend/csrs-portal/package.json]: Fake reference was removed

File: src/frontend/csrs-portal/src/assets/i18n/en.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
+{
+  "app_heading": "Child Support Recalculation Service",
+  "landing": {
+    "description1": "The online tool for submitting and resolving your BC Violation Ticket disputes.",
```
[the remark this answers is not in the document]
Comment (skiselev1) [reply src/frontend/csrs-portal/src/assets/i18n/en.json]: it was cleaned

File: src/frontend/csrs-portal/README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ The following list includes the required software needed to run the application,
 
 [Download](https://git-scm.com/downloads) and install the Git version control system, and optionally [download](https://www.gitkraken.com) and install the free GitKraken Git GUI client.
 
-Clone the Traffic Court Online repository into a project directory GitKraken or the terminal by typing:
+Clone the CSRS repository into a project directory GitKraken or the terminal by typing:
```
Comment (KyleKayfish) [inline src/frontend/csrs-portal/README.md:15]: Readme does not need to be so granular to specify a Git browser or how to clone.  Can simply state clone repo. 
However as this is a public repo it should be forked

**Comment by skiselev1** [comment]
We don't need to merge from main to repo-mountie/add-compliance. This PR was created in error.

**Pull request closed without merging**